opencog/cython/PythonEval.cc

```diff
@@ -415,8 +414,7 @@ Handle PythonEval::apply(const std::string& func, Handle varargs)
     // Py_DECREF(pyDict);
 
     // If we can't find that function then throw an exception.
-    if (!pyUserFunc)
-    {
+    if (!pyUserFunc) {
```

Comment:
Though my absolute preference is to do:

```
if (some_test)
    {
    function_goes_here();
    and_more_here();
    }
else
    {
    different_function_goes_here();
    and_more_here();
    }
```
because it's easier to visually match the braces with the code and to see when there is an `else` clause.

So my personal order of preference is the above, then this:

```
if (some_test)
{
    function_goes_here();
    and_more_here();
}
else
{
    different_function_goes_here();
    and_more_here();
 }
```
and then this:
```
if (some_test) {
    function_goes_here();
    and_more_here();

} else {
    different_function_goes_here();
    and_more_here();
 }
```